source/beamline/CLS/CLSBeamlineStatus.cpp

```diff
@@ -10,18 +13,20 @@ CLSBeamlineStatus::CLSBeamlineStatus(const QString &name, QObject *parent) :
 	monoMaskControlSet_ = new AMControlSet(this);
 
 	// Setup the basic value options.
-	addOption(On, "On", true);
-	addOption(Off, "Off", true);
+	addOption(On, "Beam is ON", true);
+	addOption(Off, "Beam is OFF", true);
+
+	connect(this, SIGNAL(connected(bool)), this, SLOT(updateBeamlineStatus()));
```

Comment:
basically, CLSBeamlineStatus is a group of controls. The signals (`connected()`/`valueChanged()`) of any control managed by CLSBeamlineStatus will trigger CLSBeamlineStatus to emit the same signal.

In this way, connect to our own signals will avoid the mistake of missing a signal from the child controls.